src/qt/guiutil.cpp

```diff
@@ -902,4 +902,15 @@ QString MakeHtmlLink(const QString& source, const QString& link)
         QLatin1String("<a href=\"") % link % QLatin1String("\">") % link % QLatin1String("</a>"));
 }
 
+void PrintSlotException(
+    const std::exception* exception,
+    const QObject* sender,
+    const QObject* receiver)
+{
+    std::string description = sender->metaObject()->className();
```

Comment:
In commit "qt: Add GUIUtil::ExceptionSafeConnect function" (f7e260a471010e2d656fbc5ea8c310f6d94c26b9)

Not important, but just in case you revisit this, I think it might be a little clearer to call this variable "location" instead of "description". (I was a little confused at first about why it didn't seem to include information from the exception.)